azure-pipelines-cloud-db-integration.yml

```diff
@@ -4,83 +4,83 @@ stages:
       vmImage: 'ubuntu-latest'
 
     jobs:
-      - job: bigquery_performance_test
-        timeoutInMinutes: 30 # this should be more than sufficient since the performance typically runs < 5 min
-        variables:
-          python.version: '3.8'
+       - job: bigquery_performance_test
```

Comment:
Are these changes just pre-commit fixing the whitespace issues?

Reply:
yes 😅 planning on reverting them.